profiler/include/profile_reduce_impl.hpp

```diff
@@ -313,7 +314,9 @@ bool profile_reduce_impl_impl(bool do_verification,
             ReductionHost<InDataType,
                           AccDataType,
                           OutDataType,
-                          ReduceOpId,
+                          ReduceOperation,
+                          InElementwiseOperation_0,
```

Comment:
Just wonder why you names it after InElementwiseOperation_0 instead of InElementwiseOperation.
I do not seen InElementwiseOperation_1.

Reply:
Yes,  this is a left issue by last refactoring of the codes.   But don't want to make change in this P.R

Reply:
This is a minor issue, and it will be a new introduced into develop if not fixed by this PR

Please fix the issue in this PR